IAGrim/UI/Tabs/ModsDatabaseConfig.cs

```diff
@@ -140,6 +141,12 @@ public void ForceDatabaseUpdate(string location, string modLocation)
                 var mod = listViewMods.SelectedItems[0].Tag as ListViewEntry;
                 var entry = lvi.Tag as ListViewEntry;
 
+                if (mod != null)
+                {
+                    // Load selected mod icons
+                    ThreadPool.QueueUserWorkItem((m) => ArzParser.LoadSelectedModIcons(mod.Path));
```

Comment:
Doing it parallel to the database parse will most likely cause an OOM on lower end PCs.

Reply:
Any suggestions to an alternative?

Reply:
As per discord: Will see how it turns out.